packages/orbit-components/src/Popover/components/ContentWrapper.tsx

```diff
@@ -297,60 +102,146 @@ const PopoverContentWrapper = ({
+          "lm:transition-[opacity] lm:duration-fast ease-in-out",
+          "lm:rounded-normal",
+          "lm:shadow-raised",
+          "lm:max-h-none",
+          isInsideModal ? "lm:z-[1000]" : "lm:z-[var(--zIndex)]",
+          width ? "lm:w-[var(--width)]" : "lm:w-auto",
+          shown ? "lm:opacity-100" : "lm:opacity-0",
+        )}
+        style={cssVars}
       >
-        <StyledPopoverContent ref={content} shownMobile={shown}>
-          <StyledContentWrapper
-            actionsHeight={actionsHeight}
+        <div
+          ref={content}
+          className={cx(
+            shown ? "translate-y-0" : "translate-y-full",
+            "will-change-transform",
+            "duration-fast transition-[opacity,transform] ease-in-out",
+            "lm:transform-none",
+            "lm-transition-none",
+          )}
+        >
+          <div
             ref={scrollingElementRef}
-            windowHeight={windowHeight}
-            maxHeight={maxHeight}
+            className={cx(
+              "overflow-auto",
+              "rounded-t-[theme(spacing.sm)]",
+              "absolute left-0",
+              "w-full",
+              "bg-white-normal",
+              "max-h-[calc(var(--window-height)-var(--actions-height)-32)]",
+              "bottom-[var(--actions-height)]",
+              noPadding ? "p-0" : "p-md",
+              "lm:max-h-[var(--max-height)]",
+              "lm:rounded-normal",
+              "lm:bottom-auto",
+              "lm:left-auto",
+              "lm:relative",
+            )}
+            style={
+              {
+                "--actions-height": actionsHeight != null ? `${actionsHeight}px` : 0,
+                "--windowHeight": windowHeight != null ? `${windowHeight}px` : 0,
+                "--max-height": maxHeight != null ? `${maxHeight}px` : "100%",
+              } as React.CSSProperties
+            }
           >
-            <StyledPopoverPadding noPadding={noPadding}>{children}</StyledPopoverPadding>
-          </StyledContentWrapper>
-
+            {children}
+          </div>
           {actions ? (
-            <StyledActions ref={actionsRef}>{actions}</StyledActions>
+            <div
+              ref={actionsRef}
+              className={cx(
+                "fixed",
+                "bottom-0 left-0",
+                "w-full",
+                "box-border",
+                "p-md pt-sm",
+                "bg-white-normal",
+                "[&_.orbit-button-primitive]:w-full [&_.orbit-button-primitive]:flex-auto",
+                "lm:relative",
+                "lm:bottom-auto lm:left-auto",
+                "lm:rounded-b-normal",
+                "lm:[&_.orbit-button-primitive]:w-auto lm:[&_.orbit-button-primitive]:grow-0",
+              )}
+            >
+              {actions}
+            </div>
           ) : (
-            <StyledPopoverClose ref={actionsRef}>
+            <div ref={actionsRef} className="p-md lm:hidden lm:pb-0">
```

Comment:
```suggestion
            <div ref={actionsRef} className="p-md lm:hidden lm:invisible lm:pb-0">
```

Reply:
I know it was like that before, but it does not make sense to have it invisible and hidden at the same time. So that was the reason why I omitted that :)